NextcloudTalk/ContactsTableViewCell.m

```diff
@@ -95,4 +98,15 @@ - (void)setUserStatus:(NSString *)userStatus
     }
 }
 
+- (void)setUSerStatusMessage:(NSString *)userStatusMessage withIcon:(NSString*)userStatusIcon {
+    if ((![userStatusMessage isEqualToString:@""] || ![userStatusIcon isEqualToString:@""]) && (userStatusIcon != NULL && userStatusMessage != NULL)) {
+        NSString *statusMessage = [NSString stringWithFormat:@"%@ %@", userStatusIcon, userStatusMessage];
```

Comment:
We should only return a string with this format if there is a `userStatusIcon`.
If not, we should just return `userStatusMessage`